tests/pytest_tests/unit_tests/test_torch.py

```diff
@@ -105,3 +105,11 @@ def test_watch_bad_argument(mock_run):
         ValueError, match="log must be one of 'gradients', 'parameters', 'all', or None"
     ):
         run.watch(net, log="bad_argument")
+
+def test_log_uint8_image(mock_run, capsys):
+    run = mock_run(use_magic_mock=True)
+    uint8_tensor = torch.rand([100, 100]).to(torch.uint8)
+    im = wandb.Image(uint8_tensor)
```

Comment:
looks like you're just trying to see if this fails, no? so that there's no need to try and log it?

Reply:
Correct, we didn't mean to leave this test in. We'll remove the entire function